dependency.lic

```diff
@@ -49,10 +49,26 @@ class ArgParser
   end
 
   def display_args(data)
-    data.each do |def_set|
-      respond ''
-      respond "  ;#{Script.current.name} " + def_set.map { |x| format_item(x) }.join(' ')
-      def_set.each { |x| respond "   #{(x[:display] || x[:name]).ljust(12)} #{x[:description]} #{x[:options] ? '[' + x[:options].join(', ') + ']' : ''}"; }
+    # Display help output for script arguments
+    unless Script.current.name == "bootstrap"
+      data.each do |def_set|
+        respond ''
+        def_set.each { |x| respond " SCRIPT SUMMARY: #{x[:description]} " if x[:name].to_s == "script_summary"; }
+        respond ''
+        respond " SCRIPT CALL FORMAT AND ARG DESCRIPTIONS:"
+        respond "  ;#{Script.current.name} " + def_set.map { |x| format_item(x) }.join(' ')
+        def_set.each { |x| respond "   #{(x[:display] || x[:name]).ljust(12)} #{x[:description]} #{x[:options] ? '[' + x[:options].join(', ') + ']' : ''}" unless x[:name].to_s == "script_summary"; }
+        respond ''
+      end
+
+      # Display help output for settings used in the script. Relies on base-yaml.yaml.
+      yaml_data = get_data('yaml').to_h
```

Comment:
What do you think about `base-help.yaml` instead?

Reply:
I like this better. Will change it.